### DISCUSSION TITLE
Why I use Ubuntu - Alan Pope's blog

### DISCUSSION THREAD

**Discussion by giscus** [popey/popey.com-blog#19] [General]
# Why I use Ubuntu - Alan Pope's blog

It’s Sunday afternoon, and I’m cooking the family dinner, so I’m also listening to a podcast. I just listened to the latest episode of Linux Downtime. In it, Amolith, Gary and Joe discuss why they use the Linux distributions they do. While the food cooks, I thought I’d take 20 minutes to bang out a blog post mulling why I (still) use Ubuntu.
However, this turned more into a bit of a trip down memory lane and certainly didn’t get banged out in 20 mins.

https://popey.com/blog/2023/08/why-i-use-ubuntu/

**Comment by astrogenealogist** [comment]
Thank you Alan for your time and honesty.  - Craig Rhombs (a retired guy in the USA)

**Reply by ghost** [reply]
>as they kept the pwc kernel driver in the distro, no recompilation needed by me.

there was no dkms in 2005?

**Reply by popey** [reply]
Correct. DKMS was started by Dell in [May 2007](https://github.com/dell/dkms/commit/25fddac94745facd5ff62d2c6d2b0fc1672d60e3).

**Reply by popey** [reply]
Interestingly Wikipedia says it was made in 2003. But the Ubuntu changelog shows it didn't come in from Debian until late 2007 at version 2.0.17.4-0ubuntu1 via launchpad [bug 121676](https://pad.lv/121676). I imagine it was used internally for a while at Dell before they released it.

**Reply by ghost** [reply]
very interesting! thank you! (sorry for posting in the wrong thread, i'm not used to this commenting platform)

**Comment by Svin-kun** [comment]
Using Ubuntu since 2008. Stayed on it, when others said: I can't stand new $featurename , I'm leaving. But even I'm sick of such innovations like new intstaller, AFAIR also packaged on snap. Now I'm still on Ubuntu. I have time, I will think a lot.
